src/gridDynMain/CMakeLists.txt

```diff
@@ -16,9 +16,10 @@ add_executable(gridDynMain ${gridDynMain_sources} ${gridDynMain_headers})
 
 set(GRIDYNMAIN_BINARY_LOC ${CMAKE_CURRENT_BINARY_DIR} PARENT_SCOPE)
 
-message(STATUS "execOnly:${executable_only_components}")
+message(STATUS "execOnly:${executable_only_components}") # TODO what even are these?
 
-target_link_libraries(gridDynMain ${executable_only_components} runner)
+# TODO why wasn't fmiBuilder required before?
+target_link_libraries(gridDynMain ${executable_only_components} runner fmiBuilder)
```

Comment:
This is really confusing. Travis says `/usr/bin/ld: cannot find -lfmiBuilder`, but my environment says
```
/bin/ld: error: undefined symbol: griddyn::fmi::fmuBuilder::fmuBuilder(std::__1::shared_ptr<griddyn::gridDynSimulation>)
>>> referenced by gridDynMain.cpp
>>>               CMakeFiles/gridDynMain.dir/gridDynMain.cpp.o:(main)

/bin/ld: error: undefined symbol: griddyn::fmi::fmuBuilder::Initialize(int, char**)
>>> referenced by gridDynMain.cpp
>>>               CMakeFiles/gridDynMain.dir/gridDynMain.cpp.o:(main)

/bin/ld: error: undefined symbol: griddyn::fmi::fmuBuilder::MakeFmu(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&)
>>> referenced by gridDynMain.cpp
>>>               CMakeFiles/gridDynMain.dir/gridDynMain.cpp.o:(main)
```
if I don't have it